jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/bsm/BsmServicesController.java

```diff
@@ -0,0 +1,28 @@
+package us.dot.its.jpo.ode.bsm;
```

Comment:
TODO: VSD/ISD/BSM services controller should be merged into one, singular UDP services controller.